src/py-opentimelineio/opentimelineio/core/_core_utils.py

```diff
@@ -177,7 +177,10 @@ def __deepcopy__(self, memo):
 
                 # Hide the method frm Sphinx doc.
                 # See https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists  # noqa
-                getattr(mapClass, name).__doc__ += '\n\n:meta private:'
+                # __doc__ will be None on an embedded python interpreter where the built-in modules are provided  # noqa
```

Comment:
Last nitpick I promise: Could you change "embedded python interpreter" with "Windows embeddable package" please? Embedded in this case has multiple meaning (for example it could be a Pybind11 embedded interpreter, etc). "Windows embeddable package" will make it clear that it's really the Windows "portable" install.

Reply:
All good! :) I'll make the change